kong/cmd/utils/prefix_handler.lua

```diff
@@ -103,6 +103,33 @@ local function gen_default_ssl_cert(kong_config, target)
   return true
 end
 
+
+local function gen_trusted_certs_file(kong_config)
+  -- create SSL folder
+  local ssl_path = pl_path.join(kong_config.prefix, "ssl")
+  local ok, err = pl_dir.makepath(ssl_path)
+  if not ok then
+    return nil, err
+  end
+
+  local certs_path = pl_path.join(ssl_path, "certs.pem")
+  log.verbose("generating trusted certs file in ", certs_path)
+
+  local fd = assert(io.open(certs_path, "w+b"))
+
+  for _,path in pairs(kong_config.lua_ssl_trusted_certificate) do
```

Comment:
```suggestion
  for _, path in ipairs(kong_config.lua_ssl_trusted_certificate) do
```

`pairs` are usually used with dictionaries where keys are not integers.